website/content/docs/api-gateway/consul-api-gateway-install.mdx

```diff
@@ -168,6 +168,9 @@ The following table describes the allowed parameters for the `spec` array:
 | `consul.ports.http`               | Specifies the port for Consul's HTTP server.                                                                                                                                                                                                                                                                                                                                        | Integer | `8500`                                           |
 | `consul.scheme`                   | Specifies the scheme to use for connecting to Consul. The supported values are `"http"` and `"https"`.                                                                                                                                                                                                                                                                              | String  | `"http"`                                         |
 | `copyAnnotations.service`         | List of annotations to copy to the gateway service.                                                                                                                                                                                                                                                                                                                                 | Array   | `["external-dns.alpha.kubernetes.io/hostname"]`  |
+| `deployment.defaultInstances` | Specifies the number of instances per gateway that should be deployed by default.                                                                                                                                                                                                                                                                                                                           | Integer  | 1                                              |
+| `deployment.maxInstances` | Specifies the maximum allowed number of instances per gateway.                                                                                                                                                                                                                                                                                                                           | Integer  | 8                                              |
+| `deployment.minInstances` | Specifies the minimum allowed number of instances per gateway.                                                                                                                                                                                                                                                                                                                           | Integer  | 1                                              |
```

Comment:
Is this the best place to document absolute min and max values?

Reply:
I think it may be easier to miss updating these in prose than here if/when we change hardcoded limits.